spec/system/claims/schools/users/invite_a_user_to_a_school_spec.rb

```diff
@@ -7,180 +7,341 @@
+    expect(page).to have_summary_list_row("Email address", "barry_garlow@education.gov.uk", "Change")
+    expect(page).to have_warning_text(
+      "Barry Garlow will be sent an email to tell them you’ve added them to Shelbyville Elementary.",
+    )
+    expect(page).to have_button("Confirm and add user")
+    expect(page).to have_link("Cancel", href: "/schools/#{@shelbyville_school.id}/users")
   end
 
-  def click_back
-    click_on "Back"
+  def then_i_see_the_confirm_user_details_page_with_new_details
+    expect(page).to have_title("Claim funding for mentor training - GOV.UK")
+    expect(primary_navigation).to have_current_item("Users")
+    expect(page).to have_h1("Confirm user details")
+    expect(page).to have_h2("Details")
+    expect(page).to have_caption("User details", class: "govuk-caption-l")
+    expect(page).to have_summary_list_row("First name", "Simon", "Change")
+    expect(page).to have_summary_list_row("Last name", "Garlow", "Change")
+    expect(page).to have_summary_list_row("Email address", "simon_garlow@education.gov.uk", "Change")
+    expect(page).to have_warning_text(
+      "Simon Garlow will be sent an email to tell them you’ve added them to Shelbyville Elementary.",
+    )
+    expect(page).to have_button("Confirm and add user")
+    expect(page).to have_link("Cancel", href: "/schools/#{@shelbyville_school.id}/users")
   end
 
-  def verify_user_added
-    email_is_sent("barry.garlow@eduction.gov.uk", school)
-    visit_claims_school_users_page
-    check_user_details
+  def then_i_see_the_details_page_for_the_edited_details_user
+    expect(page).to have_title("Claim funding for mentor training - GOV.UK")
+    expect(primary_navigation).to have_current_item("Users")
+    expect(page).to have_h1("Simon Garlow")
+    expect(page).to have_caption("Shelbyville Elementary")
+    expect(page).to have_summary_list_row("First name", "Simon")
+    expect(page).to have_summary_list_row("Last name", "Garlow")
+    expect(page).to have_summary_list_row("Email address", "simon_garlow@education.gov.uk")
+    expect(page).to have_element(
+      :a,
+      text: "Remove user",
+      class: "govuk-link app-link app-link--destructive",
+    )
   end
 
-  def and_user_is_added
-    expect(page).to have_content("User added")
+  def and_i_select_add_user
+    click_on "Add user"
   end
 
-  def verify_user_added_to_another_school
-    email_is_sent("barry.garlow@eduction.gov.uk", another_school)
-    visit_another_claims_school_users_page
-    check_user_details
+  def when_i_select_back
+    click_on "Back"
   end
 
-  def email_is_sent(email, _school)
-    email = ActionMailer::Base.deliveries.find do |delivery|
-      delivery.to.include?(email) && delivery.subject == "Invitation to join Claim funding for mentor training"
-    end
+  def and_i_select_continue
+    click_on "Continue"
+  end
 
-    expect(email).not_to be_nil
+  def when_i_select_change
+    click_on "Change", match: :first
```

Comment:
This page should have `visually_hidden_text`, so you can specify which "Change" want, such as
```suggestion
    click_on "Change First name"
```
Then we don't need to worry about this test breaking if another change link is added.